code/model/DMSDocument.php

```diff
@@ -1323,4 +1338,23 @@ public function getActionTaskHtml()
 
         return $html;
     }
+
+
+    /**
+     * Removes a single or multiple "action panel" tasks
+     *
+     * @param  string|array $panelKey
+     * @return $this
+     */
+    public function removeActionPanelTask($panelKey)
```

Comment:
I think this should be single purpose. We can just call it twice and have it accept a single input each time?